NDTensors/ext/NDTensorscuTENSORExt/contract.jl

```diff
@@ -12,9 +17,54 @@ function NDTensors.contract!(
   α::Number=one(Bool),
   β::Number=zero(Bool),
 )
-  cuR = CuTensor(array(unexpose(R)), collect(labelsR))
-  cuT1 = CuTensor(array(unexpose(T1)), collect(labelsT1))
-  cuT2 = CuTensor(array(unexpose(T2)), collect(labelsT2))
+  R, T1, T2 = unexpose.((R, T1, T2))
```

Comment:
To make this code clearer let's call the arguments to the function `exposedR`, `exposedT1`, `exposedT2` and then change this line to:
```suggestion
  R, T1, T2 = unexpose.((exposedR, exposedT1, exposedT2))
```

Reply:
This has been updated. Thanks!